tests/dialects/test_cmath.py

```diff
@@ -0,0 +1,5 @@
+class Test_cmath:
```

Comment:
Should we add this in another PR?
Or should we wait for these changes?

Reply:
Can be added in other PR, yes